src/wp-includes/fonts.php

```diff
@@ -122,6 +122,17 @@ function wp_get_font_dir() {
 		'error'   => false,
 	);
 
+	wp_mkdir_p( $defaults['path'] );
```

Comment:
```suggestion
	$made_dir = wp_mkdir_p( $defaults['path'] );
```

Good catch, it will need to exist in order to check it's writable.